thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/MultiThreadView.kt

```diff
@@ -346,10 +363,15 @@ class MultiThreadView(
             JSplitPane(
                 JSplitPane.VERTICAL_SPLIT,
                 JPanel(MigLayout("ins 0, fill, flowy")).apply {
+                    val sortGroupEnumeration = sortButtons.elements
+                    add(sortGroupEnumeration.nextElement(), "split ${sortButtons.buttonCount}, flowx")
+                    for (element in sortGroupEnumeration) {
```

Comment:
The first element was popped off, so this enumeration will only have the remaining 3 - and no constraints are necessary at all